pkg/mimedb/resolve-db.go

```diff
@@ -0,0 +1,32 @@
+/*
+ * mime-db: Mime Database, (C) 2015, 2016, 2017, 2018 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package mimedb
+
+import (
+	"strings"
+)
+
+// TypeByExtension resolves the extension to its respective content-type.
+func TypeByExtension(ext string) string {
+	var contentType string
```

Comment:
default to `application/octet-stream`  such that we always return contentType default type.

Reply:
Done